packages/wear-os/internal/tasks/plain-message/message-sender-task.ts

```diff
@@ -16,15 +17,18 @@ export abstract class MessageSenderTask extends Task {
     return this._plainMessageClient;
   }
 
-  protected constructor(name: string, eventName: string) {
+  protected constructor(name: string, eventName: string, private logger = getLogger('MessageSenderTask')) {
     super(name, {
-      outputEventNames: [eventName],
+      outputEventNames: [`${name}Finished`, eventName],
     });
   }
 
   protected async onRun(taskParams: TaskParams, invocationEvent: DispatchableEvent): Promise<void | TaskOutcome> {
-    if (!areWatchFeaturesEnabled()) {
-      throw featuresNotEnabledError;
+    if (!areWatchFeaturesEnabled() || !this.plainMessageClient.enabled()) {
+      this.logger.warn('tried to send a message but watch features are not enabled!');
```

Comment:
Why not use `this.log()` instead here?

Reply:
Good question. The main reason is that I feel that message should be at warn level, and not at info level as with `this.log()`. As we discussed, it's true that with `this.log()` we also have info about task id, chain, etc... However, in my opinion, that info is not necessary because it doesn't matter which task launched this log, the important thing is the message itself.

If you see any reason why the extra info in `this.log()` would be useful, I have no problem with changing it 😉.

Reply:
No, it was just that I was wondering why. To me it looks fine as it is. 

We can think if it could be beneficial to change the `this.log()` method signature to support extra log levels (in the NTD I mean). But that's something for the future.